### ISSUE TITLE
LOST CONNECTION

### ISSUE THREAD

**Issue by liptonzuma** [MyScript/iinkJS#22]
<img width="454" alt="Screenshot 2022-12-09 at 3 22 53 AM" src="https://user-images.githubusercontent.com/61327021/206617170-d175c057-1eba-4449-b7e8-6ad636cacb26.png">
How do I get rid of this erros and also the controls does not quite work when working with maths.
`
This is my code 
 let editor = editorRef.current;

    editor = iink.register(editorRef.current, {
      recognitionParams: {
        type: 'MATH',
        protocol: 'WEBSOCKET',
        apiVersion: 'V4',
        server: {
          scheme: 'https',
          host: 'webdemoapi.myscript.com',
          applicationKey: '515131ab-35fa-411c-bb4d-3917e00faf60',
          hmacKey: '54b2ca8a-6752-469d-87dd-553bb450e9ad',
        },
        iink: {
          math: {
            mimeTypes: [
              'application/x-latex',
              'application/vnd.myscript.jiix',
              'application/mathml+xml',
            ],
          },
          export: {
            jiix: {
              strokes: true,
            },
          },
        },
      },
    });
`

**Comment by Olivier-myscript** [comment]
Dear Liptonzuma,
thank you for your post.
I am Olivier, from the technical support team at MyScript.
Were you able to solve the error message?
If not, is it possible to learn more about your use-case?
Are you using the iinkJS? What is your environment? Ideally, could we have your project/code?
Best regards,
Olivier

**Comment by liptonzuma** [comment]
Yes I did, I noticed the error was occuring as a result of react's strict mode.

**Comment by Olivier-myscript** [comment]
Ok, thank you for the update. I am glad it is working.
Best regards,
Olivier
